src/xenia/cpu/xex_module.cc

```diff
@@ -1080,7 +1084,11 @@ bool XexModule::LoadContinue() {
     switch (desc.info) {
       case XEX_SECTION_CODE:
       case XEX_SECTION_READONLY_DATA:
-        heap->Protect(address, size, kMemoryProtectRead);
+        if (cvars::writable_code_segments)
+          heap->Protect(address, size,
+                        kMemoryProtectRead | kMemoryProtectWrite);
+        else
+          heap->Protect(address, size, kMemoryProtectRead);
```

Comment:
A little bit of simplification here. Usage of ternary operator.
Instead of having if-else you can simplify it just to:

`heap->Protect(address, size,
                        cvars::writable_code_segments ? kMemoryProtectRead | kMemoryProtectWrite : kMemoryProtectRead );`

It doesn't look good on github, but after linting it should look nice